File: 1012-area.py
A,B,C=input().split(" ")
A=float(A)
B=float(B)
C=float(C)
PI=3.14159
triangulo=A*C/2.0
circulo=PI*pow(C,2)
trapezio=((A+B)*C)/2.0
quadrado=pow(B,2)
retangulo=A*B
print("TRIANGULO: {:.3f}".format(triangulo))
print("CIRCULO: {:.3f}".format(circulo))
print("TRAPEZIO: {:.3f}".format(trapezio))
print("QUADRADO: {:.3f}".format(quadrado))
print("RETANGULO: {:.3f}".format(retangulo))

File: 1012-area.c
#include <stdio.h>
#include <math.h>

#define PI 3.14159

int main( void )
{
  double A,B,C;
  double triangulo, circulo, trapezio, quadrado, retangulo;

  scanf("%lf %lf %lf", &A, &B, &C );
  
  triangulo = A*C/2.0;
  circulo = PI*pow(C,2);
  trapezio = ((A+B)*C)/2.0;
  quadrado = pow(B,2);
  retangulo = A*B;

  printf("TRIANGULO: %.3f\n", triangulo );
  printf("CIRCULO: %.3f\n", circulo );
  printf("TRAPEZIO: %.3f\n", trapezio );
  printf("QUADRADO: %.3f\n", quadrado );
  printf("RETANGULO: %.3f\n", retangulo );  
  return( 0 );
}


